app/src/main/java/com/dkanada/gramophone/service/MusicService.java

```diff
@@ -1301,25 +1303,4 @@ public void run() {
             handleAndSendChangeInternal(MEDIA_STORE_CHANGED);
         }
     }
-
-    private class ThrottledSeekHandler implements Runnable {
```

Comment:
Not really sure why this was needed...

Reply:
My guess is that every single change will trigger the intent, which will get processed by an arbitrary number of classes. That might cause older phones or maybe even current phones to run into performance issues but I'd have to test it out first.